<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerLongitudinalPETCTStudySelectionWidget</class>
 <widget class="QWidget" name="qSlicerLongitudinalPETCTStudySelectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>12</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="qSlicerLongitudinalPETCTStudySelectionTableWidget" name="TableStudySelection" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="WidgetButtonsPanel" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="ButtonVolumeRendering">
          <property name="toolTip">
           <string>Enable/disable volume rendering of the PET image data</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../qSlicerLongitudinalPETCTModule.qrc">
            <normaloff>:/Icons/VolumeRendering.png</normaloff>:/Icons/VolumeRendering.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>19</width>
            <height>19</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonSpinView">
          <property name="toolTip">
           <string>Enable/disable spinning of the PET image data volume rendering</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../qSlicerLongitudinalPETCTModule.qrc">
            <normaloff>:/Icons/ViewSpin.png</normaloff>:/Icons/ViewSpin.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>18</width>
            <height>19</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ctkDoubleSlider" name="SliderOpacityPow" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Modify the exponential function for the volume rendering opacity</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="SpinBoxOpacityPow">
          <property name="toolTip">
           <string>Modify the exponential function for the volume rendering opacity</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>3.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.500000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="LabelInfo">
         <property name="styleSheet">
          <string notr="true">QLabel QToolTip {background-color: #858585; color: #FFF;}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../qSlicerLongitudinalPETCTModule.qrc">:/Icons/help.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ctkExpandButton" name="ExpandButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Show/hide expandable volume rendering settings panel</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="mirrorOnExpand" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkExpandButton</class>
   <extends>QToolButton</extends>
   <header>ctkExpandButton.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerLongitudinalPETCTStudySelectionTableWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerLongitudinalPETCTStudySelectionTableWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkDoubleSlider</class>
   <extends>QWidget</extends>
   <header>ctkDoubleSlider.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qSlicerLongitudinalPETCTModule.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ExpandButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>WidgetButtonsPanel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
